src/shared/components/FileUpload/index.tsx

```diff
@@ -74,15 +74,19 @@ const FileUploader: React.FunctionComponent<FileUploaderProps> = ({
   >([]);
   const [fileList, setFileList] = React.useState<UploadFile[]>([]);
 
-  const handleFileUpload = async (customFileRequest: CustomFileRequest) => {
-    try {
-      const options = storageId ? { storage: storageId } : undefined;
-      const nexusFile = await onFileUpload(customFileRequest.file, options);
-      setRecentlyUploadFileList([...recentlyUploadedFileList, nexusFile]);
-      customFileRequest.onSuccess('Successfully uploaded file');
-    } catch (error) {
-      customFileRequest.onError(error);
-    }
+  const handleFileUpload = (customFileRequest: RcCustomRequestOptions) => {
```

Comment:
Same comment for this one

Reply:
version differences, I ran `yarn audit fix` its probably why